main.tf

```diff
@@ -113,31 +113,31 @@ module "container_definition" {
-    LOG_HANDLERS_MAIN_FORMATTER                                  = "json"
-    LOG_HANDLERS_MAIN_TIMESTAMP_FORMAT                           = "2006-01-02T15:04:05.999Z07:00"
-    METRIC_CLOUDWATCH_NAMING_PATTERN                             = "!nodecode {env}/{group}/{app}"
-    METRIC_ENABLED                                               = var.metric_enabled
-    METRIC_WRITER                                                = "cw"
-    REDIS_DEFAULT_ADDRESS                                        = ""
-    REDIS_DEFAULT_DIALER                                         = "srv"
-    REDIS_DEFAULT_NAMING_PATTERN                                 = "!nodecode {name}.{group}.{env}.{family}"
-    SENTRY_DSN                                                   = try(module.sentry[0].dsn, var.sentry_dsn)
-    SENTRY_ENVIRONMENT                                           = module.this.environment
-    STREAM_METRICS_MESSAGES_PER_RUNNER_CLOUDWATCH_NAMING_PATTERN = "!nodecode {env}/{group}/{app}"
-    STREAM_METRICS_MESSAGES_PER_RUNNER_ECS_SERVICE               = "{app_group}-{app_name}"
-    STREAM_METRICS_MESSAGES_PER_RUNNER_DYNAMODB_NAMING_PATTERN   = "!nodecode {env}-{modelId}"
-    TRACING_ADDR_TYPE                                            = "srv"
-    TRACING_ENABLED                                              = var.tracing_enabled
+    AWS_DEFAULT_REGION                                = module.this.aws_region
+    AWS_SDK_RETRIES                                   = 10
+    CLOUD_AWS_DEFAULTS_ENDPOINT                       = ""
+    CLOUD_AWS_DYNAMODB_CLIENTS_DEFAULT_NAMING_PATTERN = "!nodecode {env}-{group}-{modelId}"
+    CLOUD_AWS_KINESIS_CLIENTS_DEFAULT_NAMING_PATTERN  = "!nodecode {env}-{group}-{streamName}"
+    CLOUD_AWS_SNS_CLIENTS_DEFAULT_NAMING_PATTERN      = "!nodecode {env}-{group}-{topicId}"
+    CLOUD_AWS_SQS_CLIENTS_DEFAULT_NAMING_PATTERN      = "!nodecode {env}-{group}-{queueId}"
+    ENV                                               = module.this.environment
+    ENVIRONMENT                                       = module.this.environment
+    FIXTURE_GROUP_NAME                                = "$FIXTURE_GROUP_NAME"
+    LOG_HANDLERS_MAIN_FORMATTER                       = "json"
+    LOG_HANDLERS_MAIN_TIMESTAMP_FORMAT                = "2006-01-02T15:04:05.999Z07:00"
+    METRIC_CALCULATOR_ECS_CLUSTER                     = "{env}"
+    METRIC_CALCULATOR_ECS_SERVICE                     = "{app_group}-{app_name}"
+    METRIC_CALCULATOR_DYNAMODB_NAMING_PATTERN         = "!nodecode {env}-{modelId}"
+    METRIC_CLOUDWATCH_NAMING_PATTERN                  = "!nodecode {env}/{group}/{app}"
+    METRIC_ENABLED                                    = var.metric_enabled
+    METRIC_WRITER                                     = "cw"
+    REDIS_DEFAULT_ADDRESS                             = ""
+    REDIS_DEFAULT_DIALER                              = "srv"
+    REDIS_DEFAULT_NAMING_PATTERN                      = "!nodecode {name}.{group}.{env}.{family}"
```

Comment:
I think it makes sense to adjust the naming patterns in gosoline to what we see for ourselves as the default.

The way we have it now only pollutes the terraform module and the infrastructure it creates with environment variables that are not really need it when gosoline defaults are proper.

This can save us easily 9 vars.